docs/docs/50-user-guide/60-reference-docs/80-webhook-receivers/artifactory/index.md

```diff
@@ -0,0 +1,142 @@
+---
+sidebar_label: Artifactory
+---
+
+# Artifactory Webhook Receiver
+
+The Artifactory Webhook Receiver responds to `pushed` events originating from 
+Artifactory repositories by _refreshing_ all `Warehouse` resources subscribed to 
+those repositories.
```

Comment:
See https://github.com/akuity/kargo/pull/4514/files#r2196271025.

We might want to be explicit here about an event type some users might expect to be supported that we are _not_ supporting.